test/cli_network_test.go

```diff
@@ -326,7 +326,7 @@ func (suite *PouchNetworkSuite) TestNetworkPortMapping(c *check.C) {
 		"-p", "9999:80",
 		image).Assert(c, icmd.Success)
 
-	time.Sleep(1 * time.Second)
+	time.Sleep(10 * time.Second)
```

Comment:
Why to change this?

Reply:
The network test is always fail, so I increase the delay time.